docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: "3"
+
+services:
+  mysql:
+    container_name: mariadb
+    image: mariadb
+    ports:
+      - "3306"
+    environment:
+      MYSQL_ROOT_PASSWORD: mariaSql
+      MYSQL_DATABASE: opencart
+    restart: always
+  web:
+    container_name: opencart
+    image: sunilsahoo/opencart
```

Comment:
You still have this static image here

Reply:
This is a simple apache-php image. it has nothing to do with opencart.